src/update-labels-in-github.js

```diff
@@ -0,0 +1,166 @@
+const request = require('request-promise-native');
+const fs = require('fs');
+
+const validate = require('./config-validator.js');
+
+const program = require('commander');
+
+program
+    .usage('[options] <file>')
+    .option('-o, --org [org]', 'The name of the org.')
+    .option('-r, --repo [repo]', 'The name of the repo.');
+
+program.on('--help', function() {
+  console.log(`
+The following environment variables must be set:
+ - GIT_USER - the git user that owns the token specified in GIT_API_TOKEN
+ - GIT_API_TOKEN - the git API token used to access the GitHub API`);
+});
+
+program.parse(process.argv);
+
+if ((program.org && program.repo) || !(program.org || program.repo)) {
+  throw 'Either org or repo must be specified';
+}
+
+const configLocation = process.argv[process.argv.length - 1];
+const toUpdate = program.org ? { "org": program.org } : { "repo": program.repo };
+
+const user = process.env.GITHUB_USER;
+const token = process.env.GITHUB_API_TOKEN;
+
+const config = JSON.parse(fs.readFileSync(configLocation, 'utf-8'));
+
+validate(config);
+
+const opts = {
+  'headers': {
+    'User-Agent': 'ISISBusApps-OrgLabels',
+    'Accept': 'application/vnd.github.symmetra-preview+json',
+    'Content-Type': 'application/vnd.github.symmetra-preview+json'
+  },
+  'auth': {
+    'user': user,
+    'pass': token,
+    'sendImmediately': true
+  },
+  json: true
+};
+
+function readDeclaredLabelsFromConfig(config) {
+
+  const labels = [];
+
+  for (let category of config) {
+    const priorities = category['visual-priorities'];
+
+    if (!category.labels) {
+      continue;
+    }
+
+    for (let [labelName, labelMeta] of Object.entries(category.labels)) {
+      const labelPriority = labelMeta["visual-priority"];
+      const color = priorities[labelPriority].replace('#', '');
+
+      const fullName = category.name + ': ' + labelName;
+
+      labels.push({
+        name: fullName,
+        description: labelMeta.description,
+        color: color
+      });
+    }
+  }
+
+  return labels;
+}
+
+const declaredLabels = readDeclaredLabelsFromConfig(config);
+
+async function fetchReposForOrg(orgName) {
+  return await request.get(`https://api.github.com/orgs/${orgName}/repos?per_page=100`, opts);
+}
+
+async function fetchRepo(repoName) {
+  return await request.get(`https://api.github.com/repos/${repoName}`, opts);
```

Comment:
Not sure if I somehow ended up using a different version of the github api, but I could only get this working with:
`https://api.github.com/repos/<REPO_OWNER_NAME>/<REPO_NAME>`

Reply:
Can (or should) you supply the repo as isisbusapps/schedule say? It might be worth just clarifying in the readme. I agree, I've always had to specify the owner too.